services/src/main/java/io/druid/cli/ServerRunnable.java

```diff
@@ -45,5 +45,8 @@ public void run()
     catch (Exception e) {
       throw Throwables.propagate(e);
     }
+    finally {
+      lifecycle.stop();
```

Comment:
I think this line snuck in during some of my commit shuffling. I'll split it out before final rebase